api/src/datasets.py

```diff
@@ -51,7 +51,7 @@
 
 es = Elasticsearch([settings.ELASTICSEARCH_URL], port=settings.ELASTICSEARCH_PORT)
 
-tds_api = settings.TDS_API
+tds_host = settings.TDS_HOST
```

Comment:
host implies hostname, but this is a URL. We should name it to reflect what it is.

Reply:
I will make that change, I misinterpreted the comment earlier.